packages/app-store/utils.ts

```diff
@@ -132,18 +133,26 @@ export function getAppFromLocationValue(type: string): AppMeta | undefined {
   return ALL_APPS.find((app) => app?.appData?.location?.type === type);
 }
 
+export const defaultVideoAppCategories: AppCategories[] = [
+  "conferencing",
+  "messaging",
+  // Legacy name for conferencing
+  "video",
```

Comment:
Added video as well to `defaultVideoAppCategories` for handling legacy name in a consistent manner.